acceptance/acceptance_test.go

```diff
@@ -119,6 +131,23 @@ func sharedCmds(tsEnv testScriptEnv) map[string]func(ts *testscript.TestScript,
 
 			ts.Setenv(args[0], strings.TrimRight(ts.ReadFile("stdout"), "\n"))
 		},
+		"sleep": func(ts *testscript.TestScript, neg bool, args []string) {
```

Comment:
While reviewing this, @BagToad and I noodled on some ideas to avoid flakes around sleeping for arbitrary amounts of time. Both approaches required a new command. One would be a `eventuallyexec` single command which would re-exec a command repeatedly until the exit code was `0`. Then we could use `jq` to hackily cause an error based on some state e.g.:

```
gh run list --json name,status --jq 'if .[0].name != "EXPECTED" then halt_error else .[0].name end'
```

A perhaps better approach we spiked in https://github.com/cli/cli/commit/f98f45fce9e9c34abb231efe9ba9b821ca684315

Which allows for retryable stdout assertions like so:

```
retryableexec gh workflow view 'Test Workflow Name'
eventuallystdout 'Test Workflow Name - workflow.yml'
```